quadhoral-openmeteo/quadhoral_load.py

```diff
@@ -0,0 +1,40 @@
+'''Loads cleaned data into the RDS instance'''
+import logging
+from psycopg2.extras import execute_values
+from dotenv import load_dotenv
+from quadhoral_extract import get_connection
+
+
+def truncate_database() -> None:
+    '''Removes future forecasts so that they can be replaced'''
+    query = '''DELETE FROM forecast WHERE at >= CURRENT_TIMESTAMP'''
+    with get_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(query)
+            conn.commit()
```

Comment:
Nice way to update the future forecasts so we get more accurate as the time gets closer